app/scripts/modules/ecs/src/instance/details/instanceDetails.html

```diff
@@ -137,13 +139,6 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
         </dd>
       </dl>
     </collapsible-section>
-    <collapsible-section heading="Console Output" ng-if="baseIpAddress">
-      <ul>
-        <li>
-          <console-output-link instance="instance"></console-output-link>
-        </li>
-      </ul>
-    </collapsible-section>
```

Comment:
Why is this getting removed?

Reply:
good question, didn't mean to. 
glad you reviewed this :)